unit_tests/tests/trigger/resources/nb2-cranking-good.csv

```diff
@@ -980,28 +976,3 @@ Time [s],crank,cam
 9.816113833,1,1
 9.829231167,0,1
 9.829947667,1,1
-9.850056500,0,1
```

Comment:
this is a mess of noise that happens at key-off that makes the test harder to deal with (ie, cut the log off while the engine is still running happily)